guides/common/modules/proc_enabling-the-installer-managed-tftp-service.adoc

```diff
@@ -0,0 +1,61 @@
+[id="enabling-the-installer-managed-tftp-service_{context}"]
+= Enabling the installer-managed TFTP service
+
+// MARC: New module. Content is based on https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_reverting-to-internal-dns-service.adoc and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc, and adjusted for TFTP.
+// Needs to be checked if it is technically correct.
+
+If you do not have a TFTP server available in your network, you can use the installer-managed TFTP service. This feature enables you to provide a TFTP service with a low maintenance effort.
+
+You can use a backup of the answer file that was created before configuring external TFTP, or you can create a backup of the answer file.
+ifndef::orcharhino[]
+For more information about answer files, see {InstallingServerDocURL}configuring-server_{project-context}[Configuring {ProjectServer}].
+endif::[]
+
+Complete the steps on the {Project} or {SmartProxyServer} that you want to configure to manage TFTP service for the domain.
+
+
+.Prerequisites
+
+ifeval::["{context}" == "{project-context}"]
+* Ensure that the following TFTP server name information is available to you.
+
+* Use the FQDN instead of the IP address where possible in case of network changes.
+endif::[]
+
+ifeval::["{context}" == "{smart-proxy-context}"]
+* You must have the correct network name (`dns-interface`) for the DNS server.
+* You must have the correct interface name (`dhcp-interface`) for the DHCP server.
+endif::[]
+
+* Contact your network administrator to ensure that you have the correct settings.
+
+
+.Procedure
+
+. Configure {Project} or {SmartProxy} as a TFTP server:
+
+** If you have a backup of the answer file that was created before configuring a non-installer-managed TFTP service:
+
+... Restore the answer file.
+
+... Re-apply the answer file:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# {foreman-installer}
+....
+
+** If you do not have a backup answer file that was created before configuring a non-installer-managed TFTP service:
+
+... Create a backup of the answer file now.
+
+... Configure {Project} or {SmartProxy} as TFTP server:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# {foreman-installer} \
+--foreman-proxy-tftp true \
+--foreman-proxy-tftp-managed true \
+--foreman-proxy-tftp-servername 192.0.2.3
```

Comment:
Possible future improvement: This generally isn't needed because the installer can derive the IP itself in most cases. It'll have issues in multi homed proxies so I'm tempted to leave this line out and only document it as something users **may** need to override.
```suggestion
```